src/components/Books.js

```diff
@@ -1,35 +1,28 @@
+import { useSelector, useDispatch } from 'react-redux';
 import Form from './form';
 import Book from './Book';
+import { addBook, removeBook } from '../redux/books/booksSlice';
 
 const Books = () => {
-  const books = [
-    {
-      id: 1,
-      title: 'Harry Potter and the Chamber of Secrets',
-      author: 'J.K. Rowling',
-    },
-    {
-      id: 2,
-      title: 'The Originals',
-      author: 'Caroline Kaplin',
-    },
-    {
-      id: 3,
-      title: 'The Firm',
-      author: 'Harvey Specter',
-    },
-    {
-      id: 4,
-      title: 'Game of Thrones',
-      author: 'Damon Salvator',
-    },
-  ];
+  const books = useSelector((state) => state.books.books);
+  const dispatch = useDispatch();
 
+  const handleRemoveBook = (bookId) => {
+      dispatch(removeBook({id: bookId}));
+      console.log(bookId);
```

Comment:
- [ ] Console.log is a powerful tool for debugging code and is often used during the development process. However, there are a few reasons why it is not a good idea to leave console.log statements in production code:

- Security Risks: Logging sensitive information in the console can lead to security breaches. This can be especially problematic if the code is running on a client-side environment.

- Performance: Console.log statements can be resource-intensive and can slow down the performance of the code, especially if they are being called frequently.

- Code Clutter: Too many console.log statements can clutter the code, making it difficult to read and maintain. This can make it harder for other developers to understand the code, which can slow down the development process.